Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/shared/create-test-footer/create-test-footer.html

```diff
@@ -1,15 +1,18 @@
 ﻿<div class="test-create-footer clearfix">
     <div class="container">
         <ul class="steps pull-left">
-            <li class="h5 active"><a [routerLink]="['/tests/sections']">Select Sections</a></li>
+            <li class="h5" [class.active]="router.url==='/tests/sections/' + testSettings.id"><a [routerLink]="['/tests/sections',testSettings.id]">Select Sections</a></li>
             <li class="h5"><em class="material-icons">keyboard_arrow_right</em></li>
-            <li class="h5"><a [routerLink]="['/tests/questions']">Select Questions</a></li>
+            <li class="h5" [class.active]="router.url==='/tests/questions/' + testSettings.id"><a [routerLink]="['/tests/questions',testSettings.id]">Select Questions</a></li>
             <li class="h5"><em class="material-icons">keyboard_arrow_right</em></li>
-            <li class="h5"><a>Settings & Schedule</a></li>
+            <li class="h5" [class.active]="router.url==='/tests/'+testSettings.id + '/settings'"><a [routerLink]="['/tests',testSettings.id,'settings']">Settings & Schedule</a></li>  
         </ul>
         <div class="buttons-block pull-right">
-            <button type="button" class="btn" [routerLink]="['/tests']">Save & Exit</button>
-            <button type="button" class="btn btn-primary">Save & Next</button>
+          <button type="button" class="btn" *ngIf="isTestSection || isTestQuestion">Save & Exit</button>
```

Comment:
Create buttons block for all three steps. its visibility will depend upon the current component,